Gruntfile.js

```diff
@@ -63,9 +63,17 @@ module.exports = function(grunt) {
     if (grunt.option('identifier')) {
       manifest.applications = { 'gecko': { 'id': grunt.option('identifier') } };
     }
-    if (browser !== 'msedge') {
-      delete(manifest['-ms-preload']);
-    }
+    if (browser === 'msedge') {
+      manifest.icons = [
+        {"sizes":"16x16", "src": "images/icon16.png"},
+        {"sizes": "30x30", "src": "images/icon30.png"},
+        {"sizes": "50x50", "src": "images/icon50.png"},
```

Comment:
I think the docs in the README need updating with these additional image sizes for Edge

Reply:
Also does the icon set have to be set like this for Edge? Or could you just add this lot to the main set of manifest icons?

Reply:
manifoldjs, annoyingly, uses this completely different icons schema.